package.json

```diff
@@ -23,10 +23,10 @@
   "license": "MIT",
   "devDependencies": {
     "@types/jest": "^25.1.4",
-    "auto": "^9.28.0",
-    "jest": "^25.4.0",
+    "auto": "^9.36.2",
+    "jest": "^25.5.4",
```

Comment:
Isn't this up to version 26 or something? Please check all the versions. Dependabot might not be up to date because of monthly checks

Reply:
done